src/wp-includes/css/wp-embed-template-ie.css

```diff
@@ -1,19 +0,0 @@
-.dashicons-no {
```

Comment:
could we not leave this in as an empty file? or with a CSS comment that it’s deprecated?